commands/templates/addons/env/rds-postgres.yml

```diff
@@ -87,6 +87,9 @@ Resources:
     Properties:
       AutoMinorVersionUpgrade: true
       EnablePerformanceInsights: true
+      EnableCloudwatchLogsExports:
+        - postgresql
+        - upgrade
```

Comment:
What does this `upgrade` do?

Reply:
Nvm... that is to export the upgrade log.

Reply:
More info here for those that are interested: https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/USER_LogAccess.Concepts.PostgreSQL.html#USER_LogAccess.Concepts.PostgreSQL.PublishtoCloudWatchLogs